templates/zerver/help/send-group-pm.md

```diff
@@ -0,0 +1,72 @@
+# Group Private Messages
+Users can send group private messages in Zulip.
+## There are three ways to send a Group Private Message:
+### One:
```

Comment:
@andrewallen00 It should be either 1) or First:

Usually you don't write out numbers like this.